src/main/java/core/basesyntax/figure/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.figure;
+
+public class IsoscelesTrapezoid extends AbstractFigure implements Figure {
+
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * height * ((base1 + base2) / 2);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid, area: "
```

Comment:
Avoid using System.out.println for drawing figures. Consider using a method that returns the figure's description as a string, which can then be printed elsewhere in the program. This maintains separation of concerns and allows for easier testing and reuse of the drawing logic.